### DISCUSSION TITLE
Livetrak L12 ->Logic ->Blackhole->Jamulus In  Jamulus Out ->L12

### DISCUSSION THREAD

**Discussion by njc235** [ExistentialAudio/BlackHole#167] [General]
Well I seem to have pulled out all of my hair, followed as many tutorials, troubleshooting guides as I can find but I am missing something somewhere.

My L12 is my audio I/f with 14 Outs and 4 ins
I have created an Aggregate device with L12 + BH so I now have 30 outs and 20 ins
I have Logic In and out set to the aggregate device with the o/p set to 5/6 which is is the first pair of BH Outs.
Jamulus In is set to BH + L12 and out set to 3/4 which is the second pair on the USB bus to the L12.  The input to Jamulus only shows the inputs to L12 and BH and not the ouputs from it so I can't select 5/6.

I have been around so many loops trying to make this work.  Can anyone help me?

**Comment by jbuecker** [comment]
I run a similar scenario ... but use 2 instances of BH 16 (One instance acting as an INPUT 1-16) the other instance acting as an Output BH (1-16) .... Aggregated inside of logic.  Im happy to try and connect to sort this out with you....I dont use Jamulus , but I do use SoundJack and JamKazam....
~JB

**Comment by njc235** [comment]
Ji JB

All sorted now thank you.  I now use BH aggregated with my L12. I can use 2 instances of Jamulus, one for my partner singing and one for me on guitar.  Feed each or of Logic tracks to different BH pairs subsequently selected in Jamulus.  Also separate the OP from Jam to different stereo tracks on the L12 for monitoring the return feed.

Thanks for your offer.

Nick
